trucks.js

```diff
@@ -245,10 +269,16 @@ var foodTrucks = [
     schedule: []
   }
 ];
+function filterTrucksByDay(day){
+  day = day[0].toUpperCase() + day.slice[1].toLowerCase();
```

Comment:
Makes sense. Sometimes it’s hard to tell the difference between the types of brackets you’ve used when reviewing the lecture or homework videos.

Thanks!

From: jamdon 
Sent: Friday, May 13, 2016 2:51 AM
To: JSCRIPT300-Spring2016/Module_3 
Cc: seancava ; Author 
Subject: Re: [JSCRIPT300-Spring2016/Module_3] Mod 3 Commit (#16)

In trucks.js:

> @@ -245,10 +269,16 @@ var foodTrucks = [
>      schedule: []
>    }
>  ];
> +function filterTrucksByDay(day){
> +  day = day[0].toUpperCase() + day.slice[1].toLowerCase();
You can index into day like an array because it's a string. The slice method you're calling on day, however, is a function, not an array, so this line of code is just going to cause an error.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub